package.json

```diff
@@ -67,15 +65,5 @@
     "transformIgnorePatterns": [
       "node_modules/(?!(@glennsl/bs-jest|bs-fetch|wonka|bs-platform)/)"
     ]
-  },
-  "lint-staged": {
-    "*.{re,rei}": [
-      "bsrefmt --in-place"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
Unfortunately, [refmt no longer has support for in-place modification 😭](https://forum.rescript-lang.org/t/reformat-rescript-in-place/207) This means we can't do any sort of pre-commit or pre-merge checks and instead just need to ask people to use the VSCode extension or run refmt themselves :oof:

Reply:
We can likely just restore support by writing a small script 👍